config/vendor_api/v1.1.yml

```diff
@@ -493,7 +493,7 @@ paths:
       summary: Create note
```

Comment:
This opens a small tin of worms about identity of users in SRSs who are now able to perform actions as though they had accounts on Manage. To be cautious, could we require `email_address`? Or even reuse the metadata field? Or perhaps even for now, descope creation and only allow reads?

Reply:
Plan here is to use the attribution part of MetaData, which is required for POST requests. I think this enforces  the presence of an email address. We've been talking about refactoring notes to have a polymorphic association with authors, supporting both `ProviderUser` and `VendorAPIUser` authors.

Allowing the creation of Notes is probably a good thing, as it would enable more information sharing between people using Manage and people using SRS systems, I think.

Reply:
It is a good thing, but I think the identity q still stands. 2 qs, I guess:

1. what's the point of including a "name" field if we're relying on metadata?
2. as far as I know many providers are using a generic address in the metadata field — does including optional free-text "name" field meaningfully mitigate that?

Reply:
Ah, I see the confusion. When creating a note via the API, vendors are supposed to pass a `CreateNote` object, which only contains a `message` value. Our code reads the metadata, creates/updates a VendorAPIUser and assigns them as the author of the note. When their system retrieves this application later (or immediately, actually, as we respond with the updated application), this contains a `notes:` section consisting of an array of Note objects, each containing a free-text `author` field, derived from the VendorAPIUser. So, `author` is read-only/part of the response.

Reply:
To answer 2, we are not sure what to do about generic addresses in the metadata. The only impact is within Manage, in places like timeline and the activity log, if they are using both Manage and an SRS. And I guess it'd be easy to spot what is going on after looking at a few notes.

Reply:
ah great, makes sense. Thanks!

Reply:
worth mentioning we did raise this with some providers (generic emails etc), and some wanted to take it away